content/brand/virtual-background.md

```diff
@@ -0,0 +1,53 @@
+
+backgrounds:
+    - name: This Is Fine
+      image: /images/virtual-backgrounds/this-is-fine-pulumipus.jpg
+      description: |
+        For those days when nothing seems to be going right and every time you try and "fix" something you are just starting another fire. Fear not, the Pulumipus is here to comfort you and your co-workers — this is fine.
+
+    - name: Maximum Chill
+      image: /images/virtual-backgrounds/maximum-chill-pulumipus.jpg
+      description: |
+         Life is hard and sometimes we just have to let everyone know that it’s time to relax and take a load off. Whether you've had an intense week of work or your kid has been asking you for 72 hours straight when the pandemic will be over, this background lets everyone know you're mentally ready to hit the beach and leave your cares behind.
+
+    - name: The Suit
+      image: /images/virtual-backgrounds/the-suit-pulumi.png
+      description: |
+         Virtual backgrounds are the wardrobe of the remote working world. While it's fun to have silly backgrounds, some calls are going to be more important than others and your virtual background needs to rise to that occasion. The subtle blurring of the foyer in the Pulumi Office provides the perfect vibe to let everyone know that today you mean business.
+
+    - name: Shred the Gnar
+      image: /images/virtual-backgrounds/shred-the-gnar-pulumipus.jpg
+      description: |
+        Everyone has those super productive days where you wake up ready to attack the world. If you woke up this morning ready to "shred the gnar,” then this background is for you. Let all your co-workers know that today is that day. It’s your day. And you are finally going to finish that thing you've been saying you’ll finish for the last 6 months.
+
+    - name: On Cloud Nine
+      image: /images/virtual-backgrounds/cloud-nine-pulumipus.jpg
+      description: |
+        Have you ever thought, “Wow I am so happy right now. I wish I had a virtual background to express how happy I am right now.” Well you’re in luck because now you do. The image of the Pulumipus gracefully flying around the city is the perfect way to express that you’re “on cloud nine.”
+
+    - name: Robot Repair
+      image: /images/virtual-backgrounds/robot-repair-pulumipus.png
+      description: |
+        Regardless of where you fall on the love/hate spectrum of robots we can all agree on one thing – they require maintenance. This background is for the days you spend grinding away on those important maintenance tasks that no one ever thanks you for, but which prevent your organization from collapsing entirely.
+
+    - name: Bring The Cheer
+      image: /images/virtual-backgrounds/bring-the-cheer-pulumipus.jpg
+      description: |
+        We might be biased but Pike’s Place Market in Seattle is one of the most elegant and cheerful places to be spending your Christmas season. You’re sure to brighten up the spirits of everyone on the call when you arrive with this background.
```

Comment:
Maybe since this holiday's passed, something more general/wintery?

Reply:
👍